### DISCUSSION TITLE
如何使用操作系统的图标资源？

### DISCUSSION THREAD

**Discussion by tsctsc6** [BYJRK/DotNet-Discussions#260] [WPF 开发]
我想在WPF中使用诸如打开文件、保存、新建文件、复制等常用功能的图标，能不能直接使用操作系统提供的图标资源（如果有的话），而不用建立一个resources文件夹，然后去网上搜索一些图标了。

**Comment by tsctsc6** [comment]
具体的说，就是能否直接使用下图的这些图标呢
![image](https://github.com/BYJRK/DotNet-Discussions/assets/93368014/0f34e796-84fb-4e37-a19b-c5e5d12c8694)

**Comment by CodingOctocat** [comment]
- 这个是 VS 的图标:
[Visual Studio Image Library](https://www.microsoft.com/en-us/download/details.aspx?id=35825)

- 这个是 Win Shell32 的图标:
[shell32.zip](https://github.com/BYJRK/DotNet-Discussions/files/14633358/shell32.zip)

**Comment by tsctsc6** [comment]
thanks
